### ISSUE TITLE
[Machine Learning] Multiclass image classification with MobileNetV2 and TensorFlow

### ISSUE THREAD

**Issue by kelvinkimani501** [section-engineering-education/engineering-education#6737]
#### Proposed title of article
[Machine Learning] Multiclass image classification using Mobile Net

#### Proposed article introduction
Image classification is the process of categorizing and labeling groups of pixels or vectors within an image based on specific rules. A model is trained on a set of image datasets so that it can be able to classify images into pre-defined labels. Multiclass image classification is a common task in computer vision, where we categorize an image into three or more classes.

MobileNetV2 is a type of convolutional neural network designed for mobile and embedded vision applications. They are based on a streamlined architecture that uses depthwise separable convolutions to build light weight deep neural networks that can have low latency for mobile and embedded devices without compromising the accuracy of the results.

MobileNetV2 is a pre-trained model. Pre-trained models are the deep learning models which are trained on very large datasets, developed, and are made available by other developers who want to contribute to this machine learning community to solve similar types of problems. We will use the pre-trained Mobile Net model to build a plant disease prediction model. 

#### Key takeaways 
1. Deep Learning pre-trained model.
2. Concept of transfer learning.
3. Downloading a pre-trained MobileNet-v2 from the TensorFlow hub.
4.  Image Preprocessing in Python (Image standardization/normalization, image scaling)
5. Build a plant disease prediction model. 

#### Article quality
In this tutorial, we will explain the stages involved in image Preprocessing in Python such as Image standardization/normalization and image scaling. It will give a reader a better understanding of how to handle data set with multi-class images. After following the processes the article will also explain the concept of transfer learning which has gained popularity recently when building deep learning models. Finally, using TensorFlow and the pre-trained MobileNetV2 model we will build a custom plant disease prediction model which will make accurate predictions.
#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)
